### PR TITLE
address some of the notes

### PR DESCRIPTION
**Pull request by vetinari**
which were mentioned in #74

### CODE REVIEW

**Comment by vetinari** [comment]
:+1:

File: docs/user/entities.rst [hunk 1]
```diff
@@ -7,12 +7,12 @@ Entities
...
 This can be as basic as a host, with its attributes hostname and IP; or something more complex, like a PostgreSQL sharded cluster with its identifier and set of connection strings.
 
-ZMON gives you two options for autonomation in/integration with your platform: storing entities via zmon-controller_'s entity service, or discovering them via the adapters in zmon-scheduler_.
+ZMON gives you two options for automation in/integration with your platform: storing entities via zmon-controller_'s entity service, or discovering them via the adapters in zmon-scheduler_.
 At Zalando we use both, connecting ZMON to tools like our CMDB but also pushing entities via REST API.
 
 ZMON's entity service describes entities with a single JSON document.
-Any entity must contain an ID that is unique within your ZMON deployment.
-We often use a pattern like <hostname>(:<port>) to create uniqueness at the host and application levels, but this is up to you.
+* Any entity must contain an ID that is unique within your ZMON deployment. We often use a pattern like ``<hostname>(:<port>)`` to create uniqueness at the host and application levels, but this is up to you.
```
Comment (mohabusama) [inline docs/user/entities.rst]: Are you sure those render properly?

Comment (vetinari) [reply docs/user/entities.rst]: now they do... I guess an empty line was missing

**Comment by vetinari** [comment]
:+1:

**Comment by vetinari** [comment]
:+1:

**Comment by vetinari** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**